### DISCUSSION TITLE
Extracting voice of different speakers in overlapping speech using pyannote

### DISCUSSION THREAD

**Discussion by mirsisbilgiteknolojileri** [pyannote/pyannote-audio#1277] [Q&A]
I am using Pyannote for speaker diarization. I am able to get the overlapping speech's start and end time but not able to do voice separation. Is there a way to use Pyannote for voice separation? If not, please suggest some way to do voice separation.

**Comment by hbredin** [comment]
`pyannote.audio` does not do speaker separation.
You might want to have a look at [`asteroid`](https://github.com/asteroid-team/asteroid).
